README.md

```diff
@@ -76,6 +76,20 @@ This project implements the [Scripts To Rule Them All](https://github.com/github
 * `script/setup` — Run after `script/server` to initialize local database with seed data
 * `script/update` — Run after changing code or switching branches to refresh server
 
+## Metabase
+
+The Metabase containers can be configured to the postgres database by using there settings aftering creating the containers. 
+| Setting           | Value             |
+| ----------------- | ----------------- |
+| Name              | (Your preference) |
+| Host              | db                |
+| Port              | 5432 (default)    |
+| Database name     | postgres          |
+| Database username | (leave blank)     |
+| Database passowrd | (leave blank)     |
```

Comment:
Can this config get passed into the metabase container via env from the docker-compose file?

Reply:
I did look around their docs I didn't see a way to have a default config with either a mounted config file or env variables. The only other thing I could think of is setting up an initial connection and checking in the `.db` files it creates and using some scripting up some kind of startup but I wasn't sure if that was worth the effort for this